jenkins/README.md

```diff
@@ -0,0 +1,100 @@
+# Jenkins
+
+[Jenkins](http://jenkins-ci.org/) is a pluggable continuous
+integration system. The Google team is running two Jenkins servers in GCE for
+the Kubernetes project. The post-commit instance runs continuous builds, unit
+tests, integration tests, code verification tests, and end-to-end tests on
+multiple providers using the latest commits to the Kubernetes repo from the
+master and release branches. The PR Jenkins instance runs these tests on each
+PR by a trusted contributor, it but only runs a subset of the end-to-end tests
+and only on GCE.
+
+## General flow
+The flow of the post-commit Jenkins instance:
+* Under the `kubernetes-build` job: Every 2 minutes, Jenkins polls for a batch
+  of new commits, after which it runs the `build.sh` script (in this directory)
+  on the latest tip. This results in build assets getting pushed to GCS and the
+  `latest.txt` file in the `ci` bucket being updated.
+* On trigger, and every half hour (which effectively means all the time, unless
+  we're failing cluster creation), e2e variants run, on the latest build assets
+  in GCS:
+  * `kubernetes-e2e-gce`: Standard GCE e2e.
```

Comment:
this list is woefully out of date. I'm not sure if we should modify it to indicate this is a pattern matching a lot of builds, to write that the list is non-exhaustive, or to just remove it entirely.